src/main/groovy/com/github/camork/filesystem/zstd/ZstdFile.groovy

```diff
@@ -0,0 +1,40 @@
+package com.github.camork.filesystem.zstd
+
+import com.github.camork.filesystem.IArchiveFile
+import com.github.camork.util.ArchiveUtils
+import com.github.camork.util.EntryInfo
+import io.airlift.compress.zstd.ZstdInputStream
+
+class ZstdFile implements IArchiveFile {
+    private final File file
+    private ZstdInputStream zstdInputStream
+
+    ZstdFile(File file) {
+        this.file = file
+    }
+
+    @Override
+    Map<String, ?> createEntriesInfoMap() {
+        Map entries = new HashMap<String, ?>()
+        EntryInfo root = ArchiveUtils.createRootEntry()
+        entries.put('', root)
+        entries.put('contents', new EntryInfo('contents', false, file.length(), file.lastModified(), root))
```

Comment:
Sorry for the later, It seem that the implement isn't completed, I will take over the left.